packages/labs/README.md

```diff
@@ -0,0 +1,33 @@
+# Lit Labs
+
+An experimental playground for Lit helpers and tools. Your mileage may vary.
+
+## Overview
+
+The APIs, modules, and libraries available in labs represent features that
+should be generally useful but are not yet ready for inclusion in the
+core Lit library for various reasons. They are published via the labs package
+primarily to gather user feedback.
+
+These features may change significantly and frequently, or they may be
+removed altogether. In some cases, when a feature stabilizes and proves to be
+useful, it may move into the core Lit library.
```

Comment:
"it may move into one of the core Lit packages or a new package"?

Reply:
Done.